Source/Extensions/Blazorise.QRCode/QRCode.cs

```diff
@@ -103,9 +106,16 @@ private static System.Drawing.Color GetColor( string color )
 
         private static Bitmap GetImage( string base64 )
         {
-            using var ms = new MemoryStream( Convert.FromBase64String( base64 ) );
+            try
+            {
+                using var ms = new MemoryStream( Convert.FromBase64String( base64 ) );
```

Comment:
Is there not an utility to check if string is a base64? 
Better then using the throwing mechanism as the check. But it can still stay for something actually unexpected.

Reply:
> > We have bigger problems now
> > ![image](https://user-images.githubusercontent.com/900302/201647620-021d4ea2-7b51-4734-9888-dd94ad14de78.png)
> 
> haha. Let me know if you need any help.
> 
> And let me know when it's ready to re-test

I'm researching JS libraries as a fallback for WASM. Or even for all platforms. It's really dumb from MS to remove `System.Drawing` support. Not like they don't have enough resources to have a team to do it from scratch to work on any platform 🤦‍♂️

Reply:
> Is there not an utility to check if string is a base64? Better then using the throwing mechanism as the check. But it can still stay for something actually unexpected.

I'm not aware of any.